lib/http_interceptor.dart

```diff
@@ -1,6 +1,7 @@
 library http_interceptor;
 
-export 'package:http/http.dart';
```

Comment:
I think it's best if we expose `http` package along with this package so we can avoid having to install manually `http` when using `http_interceptor`.